<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="|  | Expected Message |" tests="4" skipped="0" failures="2" errors="0" timestamp="2017-08-09T18:09:21" hostname="EPMXGUAW0210" time="0.089">
  <properties/>
  <testcase name="Given I set the custom message as &quot;&quot;" classname="|  | Expected Message |" time="0.0"/>
  <testcase name="When I set the custom message flag to &quot;on&quot;" classname="|  | Expected Message |" time="0.0"/>
  <testcase name="Then I should get the message &quot;Expected Message&quot;" classname="|  | Expected Message |" time="0.0">
    <failure message="org.junit.ComparisonFailure: expected:&lt;[Expected Message]&gt; but was:&lt;[]&gt;" type="org.junit.ComparisonFailure">org.junit.ComparisonFailure: expected:&lt;[Expected Message]&gt; but was:&lt;[]&gt;
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.adopt.mpp.test.api.tests.definition.java7.poc.SampleTest.getMessage(SampleTest.java:36)
	at ✽.Then I should get the message &quot;Expected Message&quot;(poc/scenarios/ScenarioOutlineExample.feature:11)
</failure>
  </testcase>
  <testcase name="classMethod" classname="|  | Expected Message |" time="100.17">
    <failure message="org.junit.ComparisonFailure: expected:&lt;[Expected Message]&gt; but was:&lt;[]&gt;" type="org.junit.ComparisonFailure">org.junit.ComparisonFailure: expected:&lt;[Expected Message]&gt; but was:&lt;[]&gt;
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.adopt.mpp.test.api.tests.definition.java7.poc.SampleTest.getMessage(SampleTest.java:36)
	at ✽.Then I should get the message &quot;Expected Message&quot;(poc/scenarios/ScenarioOutlineExample.feature:11)
</failure>
  </testcase>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
